### ISSUE TITLE
Chatbox, Objective Tracker, & Other UI Elements Replacement Request [Proposal]

### ISSUE THREAD

**Issue by jcdenton2k** [ForgeUI/ForgeUI#218]
TLDR:
I'd love it if ForgeUI were expanded to replace ALL the things.
(HighPriority) Replace the default ChatBox with ForgeUI aesthetic & optimization & options
(LowPriority)Replace the default QuestLog
(LowPriority)Replace the default ObjectiveTracker
(Optional) Replace other UI Elements; notify on obsoleted windows that affect disabled Carbine addons.

Long Version:
Sorta like the 'all in one' package that newbies could benefit from instead of keeping track of a dozen addons

Could easily customize by removing those modules that they don't want to use.

ChatBox replacement in particular is strongly needed due to the only alternative (that I could find) being an outdated version of some old addon from 2014.

Having the 'ForgeUI' aesthetic be more universal across the entire UI (and every element of it) would be fantastic.

Perhaps even have a message that appears when you open a window that will only affect the 'old' UI rather than ForgeUI elements.  It could be as simple as putting a red ! at the upper left and when you hover over it then it will say 'certain elements appearing here are handled by ForgeUI'.
